projects/fizzbuzz/fortran/seeley/README.md

```diff
@@ -0,0 +1,14 @@
+# fizzbuzz-fortran
+This program is a Fortran solution to the FizzBuzz problem.
+
+## Operation instructions
+Upon starting the program, you will be prompted to enter an integer number.  After typing the number, press enter.  The program will then repeat your choice and output a line for each integer from 1 to _n_ inclusive, where _n_ is your input number.
+
+Lines will read "Fizz" if the correpsonding number is divisible by 3, "Buzz" if divisible by 5, "FizzBuzz" if divisible by both 3 and 5, and the original number if none of these cases are true.
+
+## Build instructions
```

Comment:
I might place the Build Instructions above the Operation instructions just to avoid a little confusion.